src/main/java/mate/academy/bookstore/BookMapper.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.bookstore;
```

Comment:
let's move all Mapper classes into the specific package, e.g. `mapper`.

Reply:
done